lib/RemoteControl/src/App.cpp

```diff
@@ -160,27 +160,41 @@ void App::setup()
         String   mqttAddr = Settings::getInstance().getMqttBrokerAddress();
         uint16_t mqttPort = Settings::getInstance().getMqttPort();
 
-        /* Setup MQTT Server, Birth and Will messages. */
-        if (false ==
-            Board::getInstance().getNetwork().setConfig(clientId, ssid, password, mqttAddr, mqttPort, TOPIC_NAME_BIRTH,
-                                                        String(clientId + String(" Connected!")), TOPIC_NAME_WILL,
-                                                        String(clientId + String(" Disconnected!")), true))
+        /* Setup Network. */
+        NetworkSettings settings = {ssid, password, String("Zumo_AP"), String("")};

```

Comment:
The AP SSID should be derived from the robot's MAC address. In this case, the robot name can be used